### ISSUE TITLE
Could not find a package configuration file for "GTest"

### ISSUE THREAD

**Issue by karlfroldan** [microsoft/vcpkg#23398]
**Describe the bug**
CMake throws error when using `find_package(GTest CONFIG REQUIRED)`.

**Environment**
- OS: Ubuntu 20.04 (WSL 2, x64)
- vcpkg version: 2022-02-24-7efe7f2b04912a7d674f86a406f9f2711c0221c6
- gtest version: 1.11.0#3
- CMake version: 3.22

**To Reproduce**
1. ./vcpkg install gtest
2. Create a simple CMakeLists.txt file with `find_package( GTest CONFIG REQUIRED )`.

**Expected behavior**
Should find GTestConfig.cmake or gtest-config.cmake

**Failure logs** 
```
CMake Error at tests/CMakeLists.txt:19 (find_package):
  Could not find a package configuration file provided by "GTest" with any of
  the following names:

    GTestConfig.cmake
    gtest-config.cmake

  Add the installation prefix of "GTest" to CMAKE_PREFIX_PATH or set
  "GTest_DIR" to a directory containing one of the above files.  If "GTest"
  provides a separate development package or SDK, be sure it has been
  installed.
```

**Additional context**
Checked `$VCPKG_ROOT/installed/x64-linux/share/gtest`. Directory contains only `copyright` and `vcpkg_abi_info.txt`. Could this be the problem?

**Comment by karlfroldan** [comment]
It was an error on my side. I added was setting the `CMAKE_TOOLCHAIN_FILE` on the test directory's CMakeLists.txt since the main project did not use any libraries. Sorry

**Comment by ynahum** [comment]
hi,
I'm having a similar problem on WSL2 Ubuntu 20.4, 
I've installed gtest on the ubuntu sub system but I can't find the module (.cmake file) for some other third party open source build.
Where should it be? Can I copy such a file (FindGTestConfig.cmake) manually ?

Thanks,
Yair

**Comment by adeeconometrics** [comment]
Same problem on my WSL2 Ubuntu 20.02, CMake 3.16.3.

**Comment by dg0yt** [comment]
> Same problem on my WSL2 Ubuntu 20.02, CMake 3.16.3.

@adeeconometrics Open a new issue with clear steps.

**Comment by meifannao** [comment]
> > Same problem on my WSL2 Ubuntu 20.02, CMake 3.16.3.
> 
> @adeeconometrics Open a new issue with clear steps.

hi, I'm having a similar problem on WSL2 Ubuntu 20.4, how to reading the issue,can u help me

**Comment by JosiahBLB** [comment]
> It was an error on my side. I added was setting the `CMAKE_TOOLCHAIN_FILE` on the test directory's CMakeLists.txt since the main project did not use any libraries. Sorry

Just to expand on this for any new cmake users like myself, setting this variable needs to be done before the project() function, otherwise use the -DCMAKE_TOOLCHAIN_FILE=<path-to-file> when running cmake.

For me I am using vcpkg as a submodule so that path to this toolchain file won't change and therefore can use something like:
```cmake
if(NOT CMAKE_TOOLCHAIN_FILE)
  set(CMAKE_TOOLCHAIN_FILE
      "${CMAKE_CURRENT_LIST_DIR}/vcpkg/scripts/buildsystems/vcpkg.cmake"
      CACHE STRING "" FORCE)
endif()

project(my-project CXX)
```
